README.md

```diff
@@ -182,64 +191,134 @@ require("jupynium").setup({
 
 ## 🚦 Usage
 
-First, I recommend setting password on your notebook (rather than using tokens)
+There are 2 general steps to using Jupynium:
+
+1. Setup a Jupynium file
+2. Connect to the Jupynium server
+
+### Setup a Jupynium file
+
+Jupynium uses a unique file format (see the `Jupynium file format` section below). This `.ju.py` file is what you will primarily be interacting with, rather than the `.ipynb` file directly. The contents of the Jupynium file are synced to the browser notebook where it can be viewed in real-time. If you want to keep a copy of the notebook, it can be downloaded as an `.ipynb` file later.
+
+First, it's recommended to set a password on your notebook (rather than using tokens):
 
 ```console
 $ jupyter notebook password
 Enter password: 🔒
 
-$ jupyter notebook 			# leave notebook opened
+$ jupyter notebook    # leave notebook opened
+```
+
+#### If you want to start a new notebook
+
+1. Manually create a local Jupynium file called `<filename>.ju.py`
+2. Done! The rest happens after connecting to the server
+
+#### If you want to open an existing notebook
+
+There are currently 2 ways of converting an existing `.ipynb` file to a Jupynium file:
+
+**Option 1**: Use an included command line tool:
+
+```bash
+ipynb2jupy [-h] [--stdout] file.ipynb [file.ju.py]
 ```
 
-**Jupynium server stays alive as long as the browser is alive.**  
-So you can see them as the same thing in this doc.  
+**Option 2**: This method requires that you have already connected to the Jupynium server:
```

Comment:
You mentioned about "Jupynium server" without explaining what it is. It's in the next section that you can think of it as a selenium browser session, but I think we should mention that first at the very beginning of the Usage section